core/vm/suave_abis.go

```diff
@@ -273,6 +280,11 @@ var precompilesAbi = mustParseAbi(`[
               "internalType": "address[]",
               "name": "allowedPeekers",
               "type": "address[]"
+            },
+            {
+              "internalType": "string",
```

Comment:
Missing version field in Bid struct in the Suave ABI. Not a problem once generated artifacts PR lands.